README.md

```diff
@@ -7,17 +7,26 @@ The tokenizer’s role is to take a sentence or phrase and split it into smaller
 
 ## Details
 
-Meilisearch’s tokenizer is modular. It goes field by field, determining the most likely language for the field and running a different pipeline for each language.
+Charabia is modular. It goes field by field, determining the most likely language for the field and running a different pipeline for each language.
```

Comment:
What do you mean by _It goes field by field_, isn't this tokenizer only aware of strings only, not documents?

Reply:
I changed this phrase to a better one